OneSchedule.Domain/NotificationSender.cs

```diff
@@ -23,17 +21,17 @@ public NotificationSender(ITelegramBotClient bot, IRepository<EventEntity> event
 
         public async Task SendNotification()
         {
-            var actualTimeMinus30seconds = DateTime.Now.AddSeconds(-30);
-            var actualTimePlus30seconds = DateTime.Now.AddSeconds(30);
+            var actualTimeMinusDeltaTime = DateTime.Now.AddSeconds(-DeltaTime);
+            var actualTimePlusDeltaTime = DateTime.Now.AddSeconds(DeltaTime);
 
-            List<EventEntity> events = await _repository.FindAsync(e => true);
+            var events = await _repository.FindAsync(e => true);
```

Comment:
Why are you retrieving all the records from database and then filtering them in memory? Do that directly in database